draft-ietf-quic-tls.md

```diff
@@ -328,8 +328,7 @@ encryption levels:
 
 - PADDING and PING frames MAY appear in packets of any encryption level.
 
-- CRYPTO frames and CONNECTION_CLOSE frames signaling errors at the QUIC layer
-  (type 0x1c) MAY appear in packets of any encryption level except 0-RTT.
+- CRYPTO frames MAY appear in packets of any encryption level except 0-RTT.
```

Comment:
I am not sure if this change is correct, because this would mean that CONNECTION_CLOSE is allowed only in 0-RTT and 1-RTT levels.

I think what we need to do here is add CONNECTION_CLOSE (type 0x1c) to the point above that talks about PADDING and PING frames.